sources/packages/web/src/components/generic/Banner.vue

```diff
@@ -4,12 +4,12 @@
       <div class="label-bold-normal">{{ header }}</div>
     </template>
     <v-row>
-      <v-col cols="10">
+      <v-col md="10" lg="10" xl="10">
```

Comment:
If I am not wrong, once you apply one breakpoint it is valid for that one and the greater ones.
From: https://next.vuetifyjs.com/en/api/v-col/
What was the idea here removing the `cols="10"`?